src/components/Table/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import { Items } from '../Items';
+import { Pagination } from '../Pagination';
+
+type Props = {
+  view:() => string[];
+  pages: string[];
+  items: string[]
+  pageCur: number;
+  limit: number;
+  setPage: (page: number) => void;
+  setLimit: (page: number) => void;
+};
+
+export const Table: React.FC<Props> = ({
+  view,
+  pages,
+  pageCur,
+  items,
+  limit,
+  setPage,
+  setLimit,
+}) => {
+  const startItem: number = pageCur * limit + 1 - limit;
```

Comment:
There's a potential off-by-one error here. If the 'pageCur' is 1, the 'startItem' will be 1 which is correct. But if the 'pageCur' is 2, the 'startItem' will be 'limit + 1' which may cause the first item of the second page to be skipped. The correct calculation should be '(pageCur - 1) * limit + 1'.